app/models/settings.rb

```diff
@@ -0,0 +1,18 @@
+class Settings < ApplicationRecord
+  validates :key, presence: true, uniqueness: true
+  validates :value, presence: true
+
+  def self.registrations_enabled?
+    Settings.find_by(key: :REGISTRATIONS_ENABLED)&.value == "true" || false
+  end
+
+  def self.toggle_registrations
+    registrations_enabled = Settings.find_by(key: :REGISTRATIONS_ENABLED)
+    registrations_enabled ||= Settings.create(key: :REGISTRATIONS_ENABLED, value: "true")
```

Comment:
Why not `find_or_initialize_by`?

```ruby
registrations_enabled = Settings.find_or_initialize_by(key: :registrations_enabled)
registrations_enabled.value = "true"
registrations_enabled.save
```

Reply:
Rails API is big and my brain is not that big 😂 , I tried with create_by and had the error with the validations. This way seems perfect.

Reply:
Still had to verify if `value` existed before setting the default value.